### ISSUE TITLE
Create 2.3.0-beta01

### ISSUE THREAD

**Issue by jskeet** [nodatime/nodatime#1052]
This will help Glen to prototype his helper library for the Wondrous calendar, and the NodaTime.Serialization.JsonNet package can then depend on that to start code for OffsetDate and OffsetTime.

**Comment by jskeet** [comment]
We did this. I haven't updated the version history yet, that's all. Will leave open for just that.

**Comment by glittle** [comment]
This definitely helped! I've been able to publish a beta version of "WondrousNodaTime" :)

**Comment by roji** [comment]
@jskeet, I see there a 2nd beta available on nuget for 2.3.0, is there any estimate for the RTM? The Npgsql RTM is nearby and I'd like to reference 2.3.0, for the new `OffsetTime` type... I'm OK with referencing the beta if the RTM isn't far away.

**Comment by jskeet** [comment]
With a reason to release it, we can probably do it reasonably soon. I was mostly waiting to complete the OffsetTime docs, but that's now done.

This week is extremely busy, but I could probably do it before the end of May - does that work for your timeline? I'll add an issue specifically for this.

**Comment by roji** [comment]
Absolutely, thanks - my release is aligned with the Microsoft 2.1 release, which is at least end of May. I'll switch to 2.3.0-beta2 for now.
